### DISCUSSION TITLE
Need for tests on the project

### DISCUSSION THREAD

**Discussion by ktfth** [outline/outline#1421] [General] [answered]
Where the tests are settled? Have an need on a change related to [code review](https://github.com/outline/outline/pull/1420)

**Comment by tommoor** [comment, marked as answer]
Sorry, I don't understand the question. You can run tests locally with `make test`

**Reply by ktfth** [reply]
Thank you is just that

**Reply by ktfth** [reply]
I'm starting the work on some issues and that part was important for me, to check every thing essential for the process of delivery.
